index.js

```diff
@@ -34,7 +34,7 @@ function toSnake (str) {
 }
 
 function toCamel (str) {
-  return str.replace(/-[a-z]/g, letter => {
+  return str.replace(/-(.)/g, letter => {
```

Comment:
It will include spaces too. What about:

```suggestion
  return str.replace(/-\w/g, letter => {
```

Reply:
And we will need a test for that changes to avoid regressions

Reply:
Will there ever be spaces?  I never considered the possibility of spaces in a CSS name of any sort.  Are they allowed?  I assume you would want to eliminate spaces, not preserve them.  Eliminating spaces would be a separate pass.

All of these solutions preserve the whitespace.  `(.)` removes the "-", while `[a-z]` and `\w` leave the "-" intact, so I think `(.)` is preferable.  For example:
`/-(.)/g` converts "go-2- town" to "go2 town"
`/-\w/g` converts "go-2- town" to "go2- town"

As for testing, I have used this regexp in the past to do the same thing and it works well (excluding whitespace).  And I tested it again with a few examples prior to posting it.  Do you have tests in place that we can run?  What would you require from me to run tests?

If testing this is too much effort for what is not a very consequential change, please feel free to reject this PR.  I won't take offense :)  I don't think that it affects many cases and the increase in efficiency is minor.

Reply:
Oops, you need ` return str.replace(/-(\w)/g, letter => {`

Reply:
> Do you have tests in place that we can run?

Yes, just change this test https://github.com/postcss/postcss-easings/blob/main/index.test.js#L68-L72

Reply:
I just tested `/-(\w)/g` and it returns the same result as `/-\w/g`.  What did you intend it to do differently?  As I said, `(.)` is the only solution that removes the "-", and all of them preserve the post-hyphen whitespace.

I see that a series of automated Node.js tests just failed.  What does that mean?

index.test.js does not test for non-letters, do you want to add a test for that?  It seems to me that the relevant test is this block, the one above the one in your link, as it uses snake-case:
https://github.com/postcss/postcss-easings/blob/4591c4572b9117b62e5fc5f96047dc9a04f2976b/index.test.js#L62-L66

Is index.test.js run automatically?  If not, how do you run it?  I might need to learn more about how postcss works to understand how to modify it.

Reply:
> Is index.test.js run automatically?

Yes, on CI.

To run it locally:

```
pnpm install
pnpm test
```

Reply:
> I just tested /-(\w)/g and it returns the same result as /-\w/g. What did you intend it to do differently? As I said, (.) is the only solution that removes the "-", and all of them preserve the post-hyphen whitespace.

Let’s add more examples to test and see what do you want to do

Reply:
OK.  I've got to switch to some other tasks right now, but I'll get back to this tonight or tomorrow.
What about the current test failures?  From what I see in the details links they are about a lock file and have nothing to do with my changes.

Reply:
It seems to me that the first thing to do is to get this to pass the tests as they are now, then add additional cases as desired.

Reply:
> From what I see in the details links they are about a lock file and have nothing to do with my changes.

Yes, I already fixed it in `main`. You can try to rebase your changes to `main`.

Reply:
OK, rebase complete and checks now pass.  At least this PR doesn't break anything :)
If I am going to make changes to index.test.js, it will require yet another PR, which will have to be accepted prior to this one, and which might fail with the current `toCamelCase()` code.  Though the new tests won't run automatically in that PR, so that's probably OK.
I think the only tests to add are using a number or underscore and whitespace, though I still think that whitespace in a name will break something somewhere down the line, unless there is a separate pass to eliminate it or throw an error if it's there.  That would be the simplest: check for embedded whitespace and throw.
I just checked and that's exactly what you already do with `EASING_NAME` here: https://github.com/postcss/postcss-easings/blob/65dc3dfd669664b952d4edbe375d2a757398bb5c/index.js#L46
and here: https://github.com/postcss/postcss-easings/blob/65dc3dfd669664b952d4edbe375d2a757398bb5c/index.js#L52

Whitespace in the name is already eliminated prior to `toCamelCase()`.  So there's no need to add a test for it.  The only additional tests would be for "-" + number or underscore, the only non-letters allowed by `\w`.  `EASING_NAME` already uses `\w` so embedded numbers and underscores are generally allowed.

Reply:
> whitespace

Whitespaces should not be accepted in easing names

Reply:
No they should not, and index.js throws an error if the name contains whitespace prior to calling `toCamelCase()`, so this regexp will never encounter a whitespace character.  There is no issue with whitespace here.  Right?

Reply:
Don’t know. Let’s add a few test cases to see how this RegExp will be used.

Reply:
OK, I'll get back to this a bit later.  Gotta move on to something else right now.